src/components/Templates/Center.jsx

```diff
@@ -1,18 +1,127 @@
+      });
+
+      // Ordeno la info de forma descendente según la propiedad createdAt
+      const latestPosts = [...lastMonthPosts].sort((a, b) => {
+        return new Date(b.createdAt) - new Date(a.createdAt);
+      });
+
+      setFilteredPosts(latestPosts);
+    }else if (type === 'Year') {
+      // me traigo los posts creados en el último año
+      const lastYearPosts = posts.filter(post => {
+        const postDate = new Date(post.createdAt);
+        const timeDifference = currentDate - postDate;
+        const oneYearInMilliseconds = 365 * 24 * 60 * 60 * 1000; // Aproximadamente 365 días
+        return timeDifference <= oneYearInMilliseconds;
+      });
+
+      // Ordeno los posts de forma descendente según la propiedad createdAt
+      const latestPosts = [...lastYearPosts].sort((a, b) => {
+        return new Date(b.createdAt) - new Date(a.createdAt);
+      });
+
+      setFilteredPosts(latestPosts);
+    }else if (type === 'Infinity') {
+      // Me traigo los posts de forma aleatoria
+      const randomPosts = [...posts].sort(() => Math.random() - 0.5);
+      setFilteredPosts(randomPosts);
+    }
+  };
+
   return (
-    <div className="w-[40rem] ">
-      <div className="my-3">
-        <Button type="filter" >Relevant</Button>
-        <Button type="filter">Lastest</Button>
-        <Button type="filter">Top</Button>
+    <div className="w-[40rem]">
+
+    <div className="my-3 flex justify-between">
+      <div>
+        <Button type="filter" onClick={() => handleFilterClick('Relevant')}className={filterType === 'Relevant' ? 'font-bold' : ''}> Relevantes</Button>
+        <Button type="filter" onClick={() => handleFilterClick('Lastest')}className={filterType === 'Lastest' ? 'font-bold' : ''}> Últimos</Button>
+        <Button type="filter" onClick={() => handleFilterClick('Top')}
+        className={filterType === 'Top' || filterType === 'Week' || filterType === 'Month' || filterType === 'Year' || filterType === 'Infinity'? 'font-bold' : ''}>Destacados</Button>
       </div>
 
-      <CardCenter></CardCenter>
+     <div className="">
+          {showAdditionalB && (
+            <>
+              <Button type="filter" onClick={() => handleFilterClick('Week')} className={filterType === 'Week' || filterType === 'Top' ? 'font-bold' : ''}>Semana</Button>
+              <Button type="filter" onClick={() => handleFilterClick('Month')} className={filterType === 'Month' ? 'font-bold' : ''}>Mes</Button>
+              <Button type="filter" onClick={() => handleFilterClick('Year')} className={filterType === 'Year' ? 'font-bold' : ''}>Año</Button>
+              <Button type="filter" onClick={() => handleFilterClick('Infinity')} className={filterType === 'Infinity' ? 'font-bold' : ''}>Infinito</Button>
+            </>
+          )}
+        </div>
+      </div>
 
-      <CardCenter></CardCenter>
+      {filteredPosts.map(post => (
+        <CardCenter key={post.id} post={post} />
+      ))}
```

Comment:
Tómala papá????
![image](https://github.com/aluzdev/react-to/assets/147784908/638bb458-1bad-40de-b896-93272fe7e236)